lib/svg.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const { withSVG } = require('../index.js');
+
+const options = {
+    dev: true,
+    isServer: true,
+};
+
+const cleanConfig = () => ({
+    module: {
+        rules: [],
+    },
+});
+
+it('should configure a unique rule correctly with inline true', () => {
+    const webpackConfig = withSVG({ inline: true })().webpack(cleanConfig(), options);
+
+    expect(webpackConfig.module.rules).toHaveLength(1);
+    expect(webpackConfig.module.rules[0]).toMatchSnapshot();
+});
+
+it('should configure a unique rule correctly with inline default (false)', () => {
+    const webpackConfig = withSVG()().webpack(cleanConfig(), options);
+
+    expect(webpackConfig.module.rules).toHaveLength(1);
+    expect(webpackConfig.module.rules[0]).toMatchSnapshot();
+});
+
+it('should return a function if next config is a function', () => {
+    const nextConfig = {
+        webpack: jest.fn(() => 'foo'),
+    };
+
+    const webpackConfig = withSVG()(nextConfig).webpack(cleanConfig(), options);
+
+    expect(nextConfig.webpack).toHaveBeenCalledTimes(1);
+    expect(webpackConfig).toBe('foo');
+});
+
+it('should change loaders in SVG in accordance with inline option', () => {
```

Comment:
I think this test implementation is incomplete as we are only testing if the inline as `true` is different as `false`. Isn't the test above `'should configure a unique rule correctly with inline true'` already doing that? If so, I think we can remove this test.

Reply:
Hmm, the 'should configure a unique rule correctly with inline true' test only tests whether both inline *true* and *false* have correct outputs, and does not compare one to another. I think we should keep a test that wants false to be a default, and for inline to be an option that changes the output.

That said, i do think those two are the same test, and could be joined into a single test, because they will never each fail individually.